view-samples/prompt-cards/README.md

```diff
@@ -0,0 +1,85 @@
+# Prompt Cards
+
+## Summary
+This sample demonstrates how to format list items as prompt cards. The design is inspired from the [Copilot Lab](https://copilot.cloud.microsoft/en-US/prompts/all) prompt library. It features the following:
+- **Responsive tile layout:** Items are displayed in a card-based layout that adjusts to different screen sizes.
+- **Conditional Icons:** Icons change dynamically based on the value of the **Prompt Category** column.
+- **Theme Color Classes:** Uses theme color classes to ensure the format displays as intended regardless of the site's theme (light, dark, custom, etc.).
+- **Gallery View:** The formatting is designed to work with the Gallery view style.
+
+![Screenshot of the prompt cards list view formatting.](./assets/screenshot.png)
+
+
+## View requirements
+
+**Important:** This formatting depends on the Gallery view style. Make sure to set your view to Gallery before applying the formatting.
+
+|Type|Internal Name|Required|
+|---|---|:---:|
+|Single line of text|Title|Yes|
+|Multiple lines of text|Prompt|Yes|
+|Choice|Prompt Category|Yes|
+|Choice (allow multiple)|Works in|Yes|
```

Comment:
I would like it to be changed to the internal name of the column that is actually used for the formatting JSON (the internal name of the column with spaces removed) !

![image](https://github.com/user-attachments/assets/946a8e80-0bc7-4465-85d7-2382775cbfea)

The reason is that the person referring to this sample is more likely to create columns by copying the column names as described. If column names containing single-byte spaces are created, the internal name will be automatically converted to “x0020,” resulting in a difference between the column name listed in the formatting JSON and the actual internal name. This creates a risk that the sample will not work correctly.

![image](https://github.com/user-attachments/assets/981ac690-4c9a-46dc-a9b3-d71731c9bc5d)

Reply:
👍